packages/devtools-sham-modules/client/shared/components/tree.js

```diff
@@ -465,7 +478,22 @@ const Tree = module.exports = createClass({
    * Sets the state to have no focused item.
    */
   _onBlur() {
-    this._focus(0, undefined);
+    this.setState({ blurred: true });
```

Comment:
hmm - it'd be nice to avoid `blurred`... is there a way to check the blur state in `shouldComponentUpdate` and perhaps previous blur state... we do have access to the element